forms/createAccountElement.ts

```diff
@@ -0,0 +1,40 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { DefaultValues, useForm } from "react-hook-form";
+import { enum as zodEnum, object, string, TypeOf } from "zod";
+
+export const createAccountElementSchema = object({
+  name: string({ required_error: "Name is required", invalid_type_error: "Name must be a string" })
+    .min(1, "Name must be between 1 and 50 characters long")
+    .max(50, "Name must be between 1 and 50 characters long"),
+  type: zodEnum([
```

Comment:
maybe we should store this array somewhere else as const variable with unique name?